themes/aframe/layout/docs.ejs

```diff
@@ -1,39 +1,79 @@
       <% if (page.type === section.type) { activeSection = section } %>
-      <h3 class="section-title docs-section-title<%- page.type === section.type ? ' section-title-active' : ' section-title-inactive' %>"><a href="<%- page_url('docs/' + section.type + '/') %>" class="section-title-link<%= page.type === section.type ? ' section-title-link-active' : ' section-title-link-inactive' %>"><%= section.section_title %></a></h3>
 
+      <!-- Section title. -->
+      <h3 class="section-title docs-section-title<%- page.type === section.type ? ' section-title-active' : ' section-title-inactive' %>">
+        <a href="<%- page_url('docs/' + section.type + '/') %>"
+           class="section-title-link<%= page.type === section.type ? ' section-title-link-active' : ' section-title-link-inactive' %>">
+          <%= section.section_title %>
+        </a>
+      </h3>
+
+      <!-- Section links. -->
       <ol class="subnav section-subnav<%= section.show_guide ? ' numbered-subnav' : '' %>">
-        <% site.pages.find({type: section.type}).sort('order').each(function (p) { %>
+        <% docs_version_filter(site.pages.find({type: section.type}).sort('order'),
+                               activeVersion).each(function (p) { %>
           <li class="subnav-item <%= page.title === p.title ? ' subnav-item-current' : '' %>">
-            <a href="<%- page_url(p.path) %>" class="subnav-link<%= page.title === p.title ? ' current' : '' %>"><%= p.title %></a>
+            <a href="<%- page_url(p.path) %>"
+               class="subnav-link<%= page.title === p.title ? ' current' : '' %>">
+              <%= p.title %>
+            </a>
           </li>
         <% }) %>
       </ol>
     <% }) %>
   </div>
-
 </aside>
 
+<!-- Documentation. -->
 <div class="content content--structured content--<%- page.type %> copy">
   <div class="copy__wrap">
-
-    <p class="page__section"><a href="<%- page_url(activeSection.path) %>" class="borderless-link"><%= activeSection.section_title %></a></p>
+    <p class="page__section">
+      <a href="<%- page_url(activeSection.path) %>" class="borderless-link">
+        <%- activeVersion %> &gt; <%= activeSection.section_title %>
+      </a>
+    </p>
     <h1 class="page__title"><%= page.title %></h1>
 
     <%- page.content %>
 
+    <!-- Previous / Next navigation. -->
     <footer class="footer guide-links c">
-      <% site.pages.find({type: page.type, order: page.order - 1}).each(function (page) { %>
-        <span class="float-left"><a href="<%- page_url(page.path) %>" class="guide-link guide-link-left">← <%= page.title %></a></span>
+      <% docs_version_filter(site.pages.find({type: page.type, order: page.order - 1}),
+                             activeVersion).each(function (page) { %>
+        <span class="float-left">
+          <a href="<%- page_url(page.path) %>" class="guide-link guide-link-left">
+            ← <%= page.title %>
+          </a><
+        /span>
```

Comment:
this newline before actually breaks things:

![image](https://cloud.githubusercontent.com/assets/203725/15521466/d185b6aa-21c0-11e6-9811-0bdebe3bd1fe.png)